cluster/Vagrantfile

```diff
@@ -19,6 +19,11 @@ ORC_SWARM = 'swarm-mode'.freeze
 ORC_KUBEADM = 'kubeadm'.freeze
 ORC_KUBEADM_IDX = 2
 
+num_nodes = (ENV["CONTIV_NODES"] || 2).to_i
+num_masters = (ENV["CONTIV_MASTERS"] || 1).to_i
+num_workers = num_nodes - num_masters
+
+
 # Global vars
 token = 'd900e1.8a392798f13b33a4'
```

Comment:
nit: we should probably create this randomly. Maybe file a future issue for this.